Graphical editor SWTBot tests failing

## Summary

Some tests fail after the upgrade to Eclipse 2019-09. This happens on Linux, macOS, and Windows.

I have disabled the tests for now.

## Environment

* **OSATE Version**: nightly
* **Operating System**: all

From the build log:
```
testGraphicalEditor(org.osate.ge.tests.endToEnd.PrimaryEndToEndTest)  Time elapsed: 64.414 s  <<< ERROR!
org.eclipse.swtbot.swt.finder.exceptions.WidgetNotFoundException: Could not find menu item matching: with mnemonic 'Rename...'
	at org.osate.ge.tests.endToEnd.PrimaryEndToEndTest.createHardwareProject(PrimaryEndToEndTest.java:268)
	at org.osate.ge.tests.endToEnd.PrimaryEndToEndTest.testGraphicalEditor(PrimaryEndToEndTest.java:35)
Caused by: org.eclipse.swtbot.swt.finder.widgets.TimeoutException: Timeout after: 5000 ms.: Could not find menu item matching: with mnemonic 'Rename...'
	at org.osate.ge.tests.endToEnd.PrimaryEndToEndTest.createHardwareProject(PrimaryEndToEndTest.java:268)
	at org.osate.ge.tests.endToEnd.PrimaryEndToEndTest.testGraphicalEditor(PrimaryEndToEndTest.java:35)
```
Running as swtbot test I get the same exception:
```
org.eclipse.swtbot.swt.finder.exceptions.WidgetNotFoundException: Could not find menu item matching: with mnemonic 'Rename...'
	at org.eclipse.swtbot.swt.finder.SWTBotFactory.waitUntilWidgetAppears(SWTBotFactory.java:474)
	at org.eclipse.swtbot.swt.finder.SWTBot.waitUntilWidgetAppears(SWTBot.java:1)
	at org.eclipse.swtbot.swt.finder.widgets.SWTBotRootMenu.menu(SWTBotRootMenu.java:126)
	at org.eclipse.swtbot.swt.finder.widgets.SWTBotRootMenu.menu(SWTBotRootMenu.java:110)
	at org.eclipse.swtbot.swt.finder.widgets.SWTBotRootMenu.menu(SWTBotRootMenu.java:89)
	at org.osate.ge.tests.endToEnd.util.UiTestUtil.clickContextMenuOfOutlineViewItem(UiTestUtil.java:499)
	at org.osate.ge.tests.endToEnd.util.OsateGeTestCommands.renameElementFromOutlineView(OsateGeTestCommands.java:505)
	at org.osate.ge.tests.endToEnd.util.OsateGeTestCommands.createConnectionAndLayout(OsateGeTestCommands.java:388)
	at org.osate.ge.tests.endToEnd.PrimaryEndToEndTest.createHardwareProject(PrimaryEndToEndTest.java:268)
	at org.osate.ge.tests.endToEnd.PrimaryEndToEndTest.testGraphicalEditor(PrimaryEndToEndTest.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.swtbot.eclipse.core.RemotePluginTestRunner.main(RemotePluginTestRunner.java:64)
	at org.eclipse.swtbot.eclipse.core.UITestApplication.runTests(UITestApplication.java:120)
	at org.eclipse.e4.ui.internal.workbench.swt.E4Testable.lambda$0(E4Testable.java:76)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.eclipse.swtbot.swt.finder.widgets.TimeoutException: Timeout after: 5000 ms.: Could not find menu item matching: with mnemonic 'Rename...'
	at org.eclipse.swtbot.swt.finder.SWTBotFactory.waitUntil(SWTBotFactory.java:524)
	at org.eclipse.swtbot.swt.finder.SWTBot.waitUntil(SWTBot.java:1)
	at org.eclipse.swtbot.swt.finder.SWTBotFactory.waitUntil(SWTBotFactory.java:498)
	at org.eclipse.swtbot.swt.finder.SWTBot.waitUntil(SWTBot.java:1)
	at org.eclipse.swtbot.swt.finder.SWTBotFactory.waitUntil(SWTBotFactory.java:486)
	at org.eclipse.swtbot.swt.finder.SWTBot.waitUntil(SWTBot.java:1)
	at org.eclipse.swtbot.swt.finder.SWTBotFactory.waitUntilWidgetAppears(SWTBotFactory.java:468)
	... 35 more
```

--- a/ge/org.osate.ge.tests/pom.xml
+++ b/ge/org.osate.ge.tests/pom.xml
@@ -30,7 +30,7 @@ censes only apply to the Third Party Software and not any other portion of this 
 	<parent>
 		<groupId>org.osate</groupId>
 		<artifactId>ge.parent</artifactId>
-		<version>2.6.0-SNAPSHOT</version>
+		<version>2.7.0-SNAPSHOT</version>
 	</parent>
 
 	<groupId>org.osate</groupId>

--- a/ge/pom.xml
+++ b/ge/pom.xml
@@ -43,7 +43,7 @@ censes only apply to the Third Party Software and not any other portion of this 
 		<module>org.osate.ge.diagram</module>
 		<module>org.osate.ge.doc</module>
 		<module>org.osate.ge.feature</module>
-<!-- 		<module>org.osate.ge.tests</module>
- -->	</modules>
+ 		<module>org.osate.ge.tests</module>
+	</modules>
 
 </project>

